static/styles/zulip.css

```diff
@@ -2687,9 +2687,33 @@ select.inline_select_topic_edit {
     margin: 0;
 }
 
-/* Hide the up and down arrows in the Flatpickr datepicker year */
-.flatpickr-months .numInputWrapper span {
-    display: none;
+.flatpickr-calendar {
+    /* Hide the up and down arrows in the Flatpickr datepicker year */
+    .flatpickr-months .numInputWrapper span {
+        display: none;
+    }
+
+    .flatpickr-time-separator {
+        position: relative;
+        left: 5px;
+    }
+
+    .flatpickr-time input {
+        margin: 0 5px 0;
+    }
+
+    @media (width < $md_min) {
+        /* Center align flatpickr on mobile
```

Comment:
I could also add `margin-left: 50px` to show the hidden part but it doesn't go well when flatpickr is shown via compose box `<time` syntax typeahead.